.ci/resolve-kata-dependencies.sh

```diff
@@ -69,6 +70,7 @@ clone_repos() {
 	"${proxy_repo}"
 	"${runtime_repo}"
 	"${shim_repo}"
+	"${katacontainers_repo}"
```

Comment:
As @grahamwhaley notes, this list was previously sorted alphebatically... :smile: